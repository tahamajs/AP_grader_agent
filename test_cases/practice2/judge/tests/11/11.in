reserve_table 1 1
reserve_table 2 1
reserve_table 3 1
reserve_table 4 1
reserve_table 5 2
reserve_table 6 2
reserve_table 7 2
reserve_table 8 2
reserve_table 9 3
reserve_table 10 3
reserve_table 11 4
reserve_table 12 4
reserve_table 13 5
reserve_table 14 5
show_table_info 1
show_table_info 5
show_table_info 6
