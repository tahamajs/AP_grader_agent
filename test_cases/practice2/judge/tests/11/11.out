Paria sits at table 1
Amir Hossein sits at table 1
Parsa sits at table 1
Mehrad sits at table 1
Fereshte sits at table 2
Amir Akbar sits at table 2
Mahdi sits at table 2
Taha sits at table 2
Sadegh sits at table 3
Hamid sits at table 3
Shaghayegh sits at table 4
Bob sits at table 4
Ted sits at table 5
Babak sits at table 5
Table ID: 1
People at the table: Amir Hossein, Mehrad, Paria, Parsa
Table remaining capacity: 0
Waiting queue length: 0
Table ID: 5
People at the table: Babak, Ted
Table remaining capacity: 1
Waiting queue length: 0
Table ID: 6
People at the table:
Table remaining capacity: 3
Waiting queue length: 0
